app/internal/page/enter_reference_number.go

```diff
@@ -78,18 +91,18 @@ type enterReferenceNumberForm struct {
 
 func readEnterReferenceNumberForm(r *http.Request) *enterReferenceNumberForm {
 	return &enterReferenceNumberForm{
-		ReferenceNumber:    page.PostFormReferenceNumber(r, "reference-number"),
-		ReferenceNumberRaw: page.PostFormString(r, "reference-number"),
+		ReferenceNumber:    PostFormReferenceNumber(r, "reference-number"),
+		ReferenceNumberRaw: PostFormString(r, "reference-number"),
 	}
 }
 
 func (f *enterReferenceNumberForm) Validate() validation.List {
 	var errors validation.List
 
-	errors.String("reference-number", "twelveCharactersAttorneyReferenceNumber", f.ReferenceNumber,
+	errors.String("reference-number", "twelveCharactersReferenceNumber", f.ReferenceNumber,
```

Comment:
probably shouldn't change these for the attorney page

Reply:
The previous keys for CP and Attorney ref content were a bit misleading:

```
"twelveCharactersAttorneyReferenceNumber": "your 12 character reference number"
"twelveCharactersReferenceNumber": "your 12 character certificate provider reference number"
```

Made a bit of a content decision here to just standardise both to be:

```
"twelveCharactersReferenceNumber": "your 12 character reference number",
```

Reply:
cool, I had assumed they were both specific